2025-03-26 16:36:17 CST [1]: [3-1] user=,db=,app=,client= LOG:  starting PostgreSQL 15.12 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
2025-03-26 16:36:17 CST [1]: [4-1] user=,db=,app=,client= LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-03-26 16:36:17 CST [1]: [5-1] user=,db=,app=,client= LOG:  listening on IPv6 address "::", port 5432
2025-03-26 16:36:18 CST [1]: [6-1] user=,db=,app=,client= LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-03-26 16:36:20 CST [30]: [1-1] user=,db=,app=,client= LOG:  database system was interrupted; last known up at 2025-03-26 14:47:17 CST
2025-03-26 16:36:31 CST [30]: [2-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 10.06 s, current path: ./base/1/2606
2025-03-26 16:36:41 CST [30]: [3-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 20.05 s, current path: ./base/1/2670
2025-03-26 16:36:51 CST [30]: [4-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 30.07 s, current path: ./base/1/4148
2025-03-26 16:37:01 CST [30]: [5-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 40.10 s, current path: ./base/16384/2613
2025-03-26 16:37:11 CST [30]: [6-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 50.00 s, current path: ./base/4/13436_fsm
2025-03-26 16:37:21 CST [30]: [7-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 60.02 s, current path: ./base/5/2696
2025-03-26 16:37:31 CST [30]: [8-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 70.04 s, current path: ./postgresql.auto.conf
2025-03-26 16:37:31 CST [30]: [9-1] user=,db=,app=,client= LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-26 16:37:31 CST [30]: [10-1] user=,db=,app=,client= LOG:  redo starts at 0/197DFF8
2025-03-26 16:37:31 CST [30]: [11-1] user=,db=,app=,client= LOG:  invalid record length at 0/197E048: wanted 24, got 0
2025-03-26 16:37:31 CST [30]: [12-1] user=,db=,app=,client= LOG:  redo done at 0/197DFF8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.07 s
2025-03-26 16:37:31 CST [28]: [1-1] user=,db=,app=,client= LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-26 16:37:32 CST [28]: [2-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.183 s, sync=0.117 s, total=0.760 s; sync files=2, longest=0.060 s, average=0.059 s; distance=0 kB, estimate=0 kB
2025-03-26 16:37:33 CST [1]: [7-1] user=,db=,app=,client= LOG:  database system is ready to accept connections
2025-03-26 18:17:27 CST [1]: [8-1] user=,db=,app=,client= LOG:  received fast shutdown request
2025-03-26 18:17:28 CST [1]: [9-1] user=,db=,app=,client= LOG:  aborting any active transactions
2025-03-26 18:17:28 CST [1]: [10-1] user=,db=,app=,client= LOG:  background worker "logical replication launcher" (PID 33) exited with exit code 1
2025-03-26 18:17:28 CST [28]: [3-1] user=,db=,app=,client= LOG:  shutting down
2025-03-26 18:17:28 CST [28]: [4-1] user=,db=,app=,client= LOG:  checkpoint starting: shutdown immediate
2025-03-26 18:17:30 CST [28]: [5-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.440 s, sync=0.001 s, total=1.683 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB
2025-03-26 18:17:30 CST [1]: [11-1] user=,db=,app=,client= LOG:  database system is shut down
